docs/releases/3.0.4.txt

```diff
@@ -20,3 +20,7 @@ Bugfixes
   related fields or parent link fields with :ref:`multi-table-inheritance` in
   the ``of`` argument, the corresponding models were not locked
   (:ticket:`31246`).
+
+* Relaxed the system check added in Django 1.9.8 for models to reallow use of
+  the same ``db_table`` by multiple intermediary tables when database routers
+  are installed (:ticket:`31277`).
```

Comment:
This patch doesn't qualify for a backport. Please remove release notes.

Reply:
Resolved.